src/utils.c

```diff
@@ -586,7 +586,7 @@ struct option longopt_all[] = {
 	[OPT_LIST_SLOTS]   = {"list-slots", no_argument, NULL, 'P'},
 	[OPT_GET_SLOT]     = {"get-slot", no_argument, NULL, 'G'},
 	[OPT_SET_SLOT]     = {"set-slot", no_argument, NULL, 'S'},
-	[OPT_CONTROLLER]   = {"controller", required_argument, NULL, 'c'},
+	[OPT_CONTROLLER]   = {"controller-type", required_argument, NULL, 'c'},
```

Comment:
please rename OPT_CONTROLLER to OPT_CNTRL_TYPE
and cntrl to cntrl_type in struct slot_request